crates/cxx-qt-gen/src/parser/mod.rs

```diff
@@ -280,7 +306,7 @@ mod tests {
 
         assert_ne!(parser.passthrough_module, module);
 
-        assert_eq!(parser.passthrough_module.attrs.len(), 0);
+        assert_eq!(parser.passthrough_module.attrs.len(), 1);
```

Comment:
what is the attribute on the passthrough_module? As we should not be passing through `cxx_qt::bridge` i guess.

Instead we might need to change the passthrough_module to be a `passthrough_items: Vec<Item>` + `module_ident: Ident` + visibility and rebuild it in the generation with something like this?
```
#[cxx::bridge]
#module_visibility mod #module_ident {
  #(#items)*
}
```

Reply:
It's the #[cxx_qt::bridge] there, since I moved to not removing it, I think that could be one candidate where removing it is not a bad idea.

Reply:
See other comment somewhere below where instead of storing the module as passthrough maybe we have

- vector of items
- doc attributes
- module ident
- module visibility

And then rebuild it

Reply:
New commit with PassthroughMod struct should do this correctly